big_tests/tests/mam_SUITE.erl

```diff
@@ -2410,6 +2415,31 @@ pagination_last_after_id5_before_id11(Config) ->
         end,
     parallel_story(Config, [{alice, 1}], F).
 
+server_returns_item_not_found_for_before_filter_with_nonexistent_id(Config) ->
+    NonexistentID = <<"AV25E9SCO50K">>,
+    RSM = #rsm_in{max = 5, direction = 'before', id = NonexistentID},
+    StanzaID = <<"before-nonexistent-id">>,
+    server_returns_item_not_found_for_nonexistent_id(Config, RSM, StanzaID).
+
+server_returns_item_not_found_for_after_filter_with_nonexistent_id(Config) ->
+    NonexistentID = <<"AV25E9SCO50K">>,
+    RSM = #rsm_in{max = 5, direction = 'after', id = NonexistentID},
+    StanzaID = <<"after-nonexistent-id">>,
+    server_returns_item_not_found_for_nonexistent_id(Config, RSM, StanzaID).
+
+server_returns_item_not_found_for_nonexistent_id(Config, RSM, StanzaID) ->
+    P = ?config(props, Config),
+    F = fun(Alice) ->
+        rsm_send(Config, Alice,
+                 stanza_page_archive_request(P, StanzaID, RSM)),
+        Res = escalus:wait_for_stanza(Alice),
+        ct:pal("res: ~p\n", [Res]),
```

Comment:
I'll remove it.

I got used to leaving such printouts because of running CT tests with Rebar3 - it's convenient, as it allows to leave them (basically, when writing a test we know which points require special attention and put the printout accordingly), but silences outputs from succeeding tests - so no clobber it everything goes well.